### ISSUE TITLE
Firefox Sync passwords

### ISSUE THREAD

**Issue by chew-z** [swick/moz-gnome-keyring-integration#21]
I have installed this add-on and without a warning it has erased ALL passwords saved in my Firefox Sync. Well I had some really rare passwords there. 

A warning and a chance to make a backup before proceeding would be probably appreciated by most users.

**Comment by chew-z** [comment]
I had been using Old version of Firefox Sync (didn't know there is New version). So that might had been the reason. 
Anyway... This seems like I am paying the price for progress :)

**Comment by auipga** [comment]
Same here. But they came back when I disabled the extension. @chew-z have a try

**Comment by chew-z** [comment]
Yes, I know. I have retrieved my old passwords - but a warning during installation would be nice.
Anyway new Firefox Sync breaks for me when this add-on is installed. Hope it could be fixed soon as I really like this addon.

**Comment by infinity0** [comment]
Sounds like the same issue as #24
